ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/monitor_telemetry_influx.schema.yml

```diff
@@ -58,6 +61,9 @@ keys:
                 - str
             url:
               type: str
+              description: |-
```

Comment:
You can also add the `pattern` key in the schema. It will not be enforced, but will be later.

Reply:
same for the other url fields you are adding

Reply:
Fixed